pages/blog/migrating-from-rest.mdx

```diff
@@ -0,0 +1,133 @@
+import withArticle from '../../ui/blog/article';
+
+export const meta = {
+  title: 'Migrating from Schemaless REST API to GraphQL',
+  tags: ['graphql', 'rest', 'graphql-mesh'],
+  author: 'arda',
+  date: '2020-05-31',
+  description:
+    "It is really easy to migrate to GraphQL from a REST API without writing everything from scratch using GraphQL Mesh even you don't have any schema for your existing REST API.",
```

Comment:
Migrating to GraphQL from a REST API has never been easier.  With tools like GraphQL Mesh you can do it automatically, without writing code and even if you don't have any schema for your existing REST API sources.